logic/src/commonMain/kotlin/com/wire/kalium/logic/data/asset/Asset.kt

```diff
@@ -53,3 +53,7 @@ fun isDisplayableImageMimeType(mimeType: String): Boolean = mimeType in setOf(
 fun isAudioMimeType(mimeType: String): Boolean = mimeType in setOf(
     "audio/mp3", "audio/mpeg", "audio/ogg", "audio/wav", "audio/x-wav", "audio/x-pn-wav"
 )
+
+fun isVideoMimeType(mimeType: String): Boolean = mimeType in setOf(
+    "video/mp4", "video/webm", "video/3gpp", "video/mkv"
```

Comment:
is this enough to cover video formats, maybe we can replace it with check that the first part is video

Reply:
I actually wanted to do that, but I'm not sure why audio and image mime types were done this way 🤔 
These video formats are supported by Android, I took them from here: https://developer.android.com/guide/topics/media/media-formats#video-codecs